ReactSkia/components/RSkComponentRootView.cpp

```diff
@@ -8,7 +8,7 @@ namespace react {
 
 RSkComponentRootView::RSkComponentRootView(const ShadowView &shadowView)
     : RSkComponent(shadowView) {}
-
+void RSkComponentRootView::updateComponetProps(const ShadowView &newShadowView,bool forceUpdate) {}
```

Comment:
if there is no definition for this function , then empty definition can be handled in header file itself.
Than having it at two places.
Same applies to RSkComponentView::updateComponetProps

Reply:
As a reference onPaint, They wrote declaration in header file and empty definition in .cpp.